src/containers/AskMeBot/AskMeBot.module.css

```
@@ -1,149 +1,502 @@
+  width: 100%;
+  max-width: 420px;
+  margin-top: 0.5rem;
 }
 
+.SuggestionCard {
+  padding: 0.5rem 1rem;
+  border-radius: 0.75rem;
+  cursor: pointer;
+  color: #191c1a;
+  background-color: #f5f5f5;
+  font-size: 0.85rem;
```

Stylelint (17.5.0)

[error] Unexpected duplicate "font-size" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)